modules/tasks/taskClaim.js

```diff
@@ -0,0 +1,74 @@
+const Promise = require('bluebird')
+const taskUpdate = require('./taskUpdate')
+const models = require('../../models')
+const SendMail = require('../mail/mail')
+const i18n = require('i18n')
+const taskFetch = require('./taskFetch')
+
+const sendConfirmationEmail = (task, user, comments) => {
+  const approveURL = `${process.env.FRONTEND_HOST}/#/task/${task.id}/claim/${comments}`
```

Comment:
I think is fragile to use comments as an URL validation.

We should create a kind of token to validate the URL

Reply:
I made the requested changes.
Could you provide the steps/data you're using to test?